go.mod

```diff
@@ -177,12 +177,12 @@ require (
 	k8s.io/component-base v0.0.0
 	k8s.io/cri-api v0.0.0
 	k8s.io/csi-translation-lib v0.0.0
-	k8s.io/gengo v0.0.0-20190116091435-f8a0810f38af
+	k8s.io/gengo v0.0.0-20190327210449-e17681d19d3a
```

Comment:
also, `k8s.io/kube-openapi@v0.0.0-20190502190224-411b2483e503` requires `k8s.io/gengo@v0.0.0-20190128074634-0689ccc1d7d6`, so we'd need to bump our pinned version for that as well (hack/lint-dependencies.sh will indicate we're pinned to a backlevel, and `GO111MODULE=on go mod graph | grep gengo` shows which dependency requires the newer version)